ModKit/UI/CharCodeGlyphs.cs

```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
+
+namespace ModKit {
+    public static partial class Glyphs {
+        public const string CharCodeCheckOn = "x";
+        public const string CharCodeCheckOff = "o";
+        public const string CharCodeCheckEmpty = "[]";
+        public const string CharCodeDisclosureOn = "v";
+        public const string CharCodeDisclosureOff = ">";
+        public const string CharCodeDisclosureEmpty = "[]";
+        public const string CharCodeEdit = "\\";
```

Comment:
I am not sure this is clear to a casual user.  Maybe use "Edit"?

Reply:
Changed to Edit.